src/exceptions/utils.ts

```diff
@@ -50,11 +40,7 @@ const toPlatformErrorResponse = async (response: undefined | unknown): Promise<P
  * };
  * ```
  */
-export type HTTPErrCodeExceptionMap<T = LensSDKException> = Partial<Record<HTTPErrorCode, (e?: PlatformErrorResponse) => T>>;
-
-const DEFAULT_MAP: HTTPErrCodeExceptionMap = {
-  401: e => new UnauthorizedException(e?.body?.message)
-};
+export type HTTPErrCodeExceptionMap<T = LensSDKException> = Partial<Record<number, (e?: PlatformErrorResponse) => T>>;
```

Comment:
Why `HTTPErrorCode` -> `number`?

Reply:
I think we don't need to maintain a list of HTTP error code, the deleted one didn't cover all status codes in the SPEC, and axios type status code as `number`